tests/integration_tests/data_steward/utils/participant_summary_requests_test.py

```diff
@@ -194,10 +194,12 @@ def test_get_deactivated_participants(self, mock_get_session, mock_token):
 
     def test_store_digital_health_status_data(self):
         # Pre conditions
+        bq_client = BigQueryClient(self.project_id)
         fq_table = f'{self.project_id}.{self.dataset_id}.{DIGITAL_HEALTH_SHARING_STATUS}'
         self.fq_table_names.append(fq_table)
-        table = Table(fq_table,
-                      schema=get_table_schema(DIGITAL_HEALTH_SHARING_STATUS))
+        table = Table(
+            fq_table,
+            schema=bq_client.get_table_schema(DIGITAL_HEALTH_SHARING_STATUS))
         table.time_partitioning = TimePartitioning(
             type_=TimePartitioningType.DAY)
         _ = self.client.create_table(table, exists_ok=True)
```

Comment:
I see `bq_client` and `self.client` being used here. Are they the same?

Reply:
It is. No need to create another instance. Good catch.